rre-search-platform/rre-search-platform-elastic-search-impl/src/main/java/io/sease/rre/search/api/impl/ExternalElasticsearch.java

```diff
@@ -37,7 +38,8 @@
 
     @Override
     public void beforeStart(Map<String, Object> configuration) {
-        // No-op for this implementation
+        // Need RRE node instantiated to get plugin info (This does not start the node)
+        super.beforeStart(configuration);
```

Comment:
As mentioned above, if we can avoid calling the full `beforeStart` method, I think we should.